src/Console/Commands/MakeConnector.php

```diff
@@ -34,10 +39,38 @@ class MakeConnector extends MakeCommand
      */
     protected $namespace = '\Http\Integrations\{integration}';
 
+    protected function resolveStubName(): string
+    {
+        return $this->option('oauth')
+            ? 'saloon.oauth-connector.stub'
+            : 'saloon.connector.stub';
+    }
+
+    protected function getOptions()
+    {
+        return [
+            ['oauth', null, InputOption::VALUE_NONE, 'Whether the connector should include the OAuth boilerplate'],
+        ];
+    }
+
     /**
-     * The default stub
+     * Interact further with the user if they were prompted for missing arguments.
      *
-     * @var string
+     * @param  \Symfony\Component\Console\Input\InputInterface  $input
+     * @param  \Symfony\Component\Console\Output\OutputInterface  $output
+     * @return void
      */
-    protected $stub = 'saloon.connector.stub';
+    protected function afterPromptingForMissingArguments(InputInterface $input, OutputInterface $output)
+    {
+        if ($this->didReceiveOptions($input)) {
+            return;
+        }
+
+        $type = select('Should the connector support OAuth?', [
```

Comment:
Hey Ash! Would you mind using the old prompt design to keep Laravel 9 support?

Reply:
I've added this now, so it _should_ hopefully work with Laravel 9 🤞